app/helpers/dashboard_helper.rb

```diff
@@ -16,7 +16,9 @@ def dashboard_actions
        content_tag(:li, '', :class=>'widget-add') do
          widgets_to_add
        end
-     )]
+     ),
+     documentation_button
```

Comment:
It's not your fault, but the indentation's all off here, it's an odd number of spaces.  If you wouldn't mind fixing it, that'd be appreciated.